kale/task.py

```diff
@@ -295,3 +354,58 @@ def _report_permanent_failure(cls, message, exception, failure_type,
                        (failure_type, message.task_name, message.task_id,
                         exception))
         logger.error(message_str)
+
+
+class RemoteException(object):
+    """Utility class for re-raising an exception from a remote process.
+
+    Keeps the original stack trace.
+    """
+    def __init__(self, exception):
+        self.exception = exception
+        __, __, self.traceback = sys.exc_info()
+
+    def re_raise(self):
+        raise self.exception, None, self.traceback
```

Comment:
raise_from doesn't support including the traceback, so I switched to using future.utils.raise_